src/main/io/ledstrip.h

```diff
@@ -92,15 +92,15 @@ typedef enum {
 } colorId_e;
 
 typedef enum {
-    LED_MODE_ORIENTATION = 0,
-    LED_MODE_HEADFREE,
-    LED_MODE_HORIZON,
-    LED_MODE_ANGLE,
-    LED_MODE_MAG,
-    LED_MODE_BARO,
-    LED_SPECIAL,
-    LED_AUX_CHANNEL
-} ledModeIndex_e;
+    LED_FCSTATE_ORIENTATION = 0,
+    LED_FCSTATE_HEADFREE,
+    LED_FCSTATE_HORIZON,
+    LED_FCSTATE_ANGLE,
+    LED_FCSTATE_MAG,
+    LED_FCSTATE_BARO,
+    LED_FCSTATE_SPECIAL,
+    LED_FCSTATE_AUX_CHANNEL
+} ledFCStates_e;
```

Comment:
```suggestion
} ledStates_e;
```

Reply:
These states are purely related to events in the FC, for use only when the 'detailed mode' wants to indicate an FC state.  They are not more general led states, so they are identified as being specifically FC related.  I was following the naming strategy of, for instance, `ledBaseFunctionId_e` and `ledOverlayId_e`.  Hence I think this name should stay as it is.
I am open to a different identifier than 'FC' in the name, however.  Its a kind of flight_mode modifier, but actually it includes other things like Baro readings and Aux channels. I really wasn't sure exactly how to name it, but they were all flight-controller related, so I chose 'FC', despite, and I agree, it looking a bit weird.